lab_1/main.py

```diff
@@ -5,6 +5,16 @@
 
```

Comment:
do not delete functions even if you are not going to implement them
return "get_adjacent_words" and others